src/app.py

```diff
@@ -27,14 +27,17 @@
 from couchbase.management.collections import CollectionSpec
 
 # setup flask
-from flask import Flask, jsonify, request
+from flask import Flask, jsonify, request, render_template
 from flask_restx import Api, Resource, fields
 
 
 class CouchbaseClient(object):
     @classmethod
     def create_client(_, *args, **kwargs):
         self = CouchbaseClient(*args)
+        print(
+            "\n The application is initializing\n Please wait until it loads\n Refresh the page if the page is not loading automatically\n"
+        )
```

Comment:
Giving the fact that we are running inside gitpod, refreshing the page will essentially refresh the whole environment, wich is not what we want here